src/prx_host.c

```diff
@@ -642,7 +657,12 @@ static int32_t prx_host_init_from_command_line(
     printf("                                    -C options are not provided, connection \n");
     printf("                                    string is read from $_HUB_CS environment\n");
     printf("                                    variable.                               \n");
+    printf(" -F, --allow-fs-browsing            Allow clients to browse the file system.\n");
     }
+    printf(" -r, --restricted-ports range       Use this setting to limit the ports the \n");
+    printf("                                    proxy will connect to. The value is a   \n");
+    printf("                                    semicolon delimited list of ports that  \n");
+    printf("                                    can contain ranges, e.g. -P 2;4;7-19;80 \n");
```

Comment:
It currently does not work, as described here!
Using --restricted-ports=4841 allows to connect to port 4840 but not 4841 for example!